app/controllers/api/messages_controller.rb

```diff
@@ -54,6 +55,13 @@ def message_drafts
 
   private
 
+  def find_tenant_tag_by_name(tag_name)
+    tag = @tenant.tags.find_by(name: tag_name)
+    @message.destroy unless tag
```

Comment:
Toto je trosku podivna logika. Nema zmysel radsej prebehnut tie tagy vsetky predtym ako sa vytvori ten object?

Reply:
To znie fajn, upravila som